thermo_nerf/thermal_as_rgb.py

```diff
@@ -0,0 +1,20 @@
+import json
+import os
+import shutil
+import stat
+from pathlib import Path
+
+
+def thermal_as_rgb(tmp_folder: Path, original_folder: Path) -> None:
```

Comment:
Sorry one more note I just thought about. To me it makes more sense to first put the original folder and to not name your variable `tmp_folder` because it is _not_  a tempeorary folder.

Using 

```python
def thermal_as_rgb(thermal_dataset: Path, rgb_dataset: Path) -> None:
```